bundles/org.openhab.binding.openthermgateway/README.md

```diff
@@ -32,41 +32,47 @@ The configuration settings for the thing are Hostname/IP address and Port, which
 
 The OpenTherm Gateway binding supports the following channels:
 
-| Channel Type ID          | Item Type            | Description                                              | Read Only |
```

Comment:
Please remove the changes of other bindings from your PR.

Reply:
I am not sure how this ended up in my pull, tried to remove it.